lib/sass/calcite-web/patterns/_accordion.scss

```diff
@@ -19,25 +19,48 @@
   }
 
   @mixin accordion-title() {
-    @include side-nav-title();
+    @include font-size(0);
+    border: 1px solid $light-gray;
+    margin: -1px 0 0 0;
+    color: $dark-gray;
+    padding: $baseline/2;
     cursor: pointer;
     &:focus,
     &:hover {
       @include transition(all, $transition);
```

Comment:
Two things here - can we use `@include transition(all .15s ease-in-out);` for this line?
Also, if you move the transition declaration out of the hover and onto the base element's CSS, it will give you an even animation in and out. Currently it's only animating "in" on hover, not "out" as well.

Reply:
done